packages/shared/src/components/squads/InviteTextField.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {
+  forwardRef,
+  ForwardRefRenderFunction,
+  ReactElement,
+  useImperativeHandle,
+} from 'react';
+import { Button } from '../buttons/Button';
+import { TextField } from '../fields/TextField';
+import CopyIcon from '../icons/Copy';
+import { useCopyLink } from '../../hooks/useCopyLink';
+import { Squad } from '../../graphql/squads';
+
+type InviteTextFieldProps = {
+  squad?: Squad;
+  isLoading?: boolean;
+};
+
+export type InviteTextFieldHandle = {
+  copyLink: () => void;
+};
+
+const InviteText: ForwardRefRenderFunction<
+  InviteTextFieldHandle,
+  InviteTextFieldProps
+> = ({ squad, isLoading }, ref): ReactElement => {
+  const token = squad?.currentMember?.referralToken ?? '';
+  const invitation = `${squad?.permalink}/${token}`;
+  const [copying, copyLink] = useCopyLink(() => invitation);
+  useImperativeHandle(ref, () => ({
+    copyLink,
```

Comment:
@sshanzel @rebelchris I have exposed the function to outside here.

Reply:
Yeah, it shouldnt have any negative effects. Pretty useful stuff if you ask me ^_^